lib/Extractor/Candidates.cpp

```diff
@@ -752,12 +762,18 @@ class ExtractExprCandidatesPass : public FunctionPass {
 
   void getAnalysisUsage(AnalysisUsage &Info) const {
     Info.addRequired<LoopInfoWrapperPass>();
+    Info.addRequired<DemandedBitsWrapperPass>();
     Info.setPreservesAll();
   }
 
   bool runOnFunction(Function &F) {
     LoopInfo *LI = &getAnalysis<LoopInfoWrapperPass>().getLoopInfo();
-    ExtractExprCandidates(F, LI, Opts, IC, EBC, Result);
+    if (!LI)
+      report_fatal_error("getLoopInfo() failed in Harvester");
```

Comment:
I don't think "Harvester" is something we need to tell users about, please just say "getLoopInfo() failed"